initramfs-init.in

```diff
@@ -629,7 +640,7 @@ else
 	apkflags="$apkflags --update-cache"
 fi
 
-if [ "$KOPT_quiet" = yes ]; then
+if [ "$KOPT_quiet" = yes ] || [ -n "$KOPT_ssh_key" ] || [ -n "$KOPT_ssh_pass" ]; then
```

Comment:
Why is this needed?

Reply:
Alright this looks more like a s390x thing rather than SSH thing. I will come up with a way to make this work. Remove this now.